src/helm/benchmark/presentation/summarize.py

```diff
@@ -1163,18 +1210,16 @@ def write_scenario_spec_instance_ids_json(self, file_path) -> None:
                 for scenario_spec_instance_ids in all_scenario_spec_instance_ids
             )
 
-
-def symlink_latest(output_path: str, suite: str) -> None:
-    # Create a symlink runs/latest -> runs/<name_of_suite>,
-    # so runs/latest always points to the latest run suite.
-    runs_dir: str = os.path.join(output_path, "runs")
-    suite_dir: str = os.path.join(runs_dir, suite)
-    symlink_path: str = os.path.abspath(os.path.join(runs_dir, LATEST_SYMLINK))
-    hlog(f"Symlinking {suite_dir} to {LATEST_SYMLINK}.")
-    if os.path.islink(symlink_path):
-        # Remove the previous symlink if it exists.
-        os.unlink(symlink_path)
-    os.symlink(os.path.abspath(suite_dir), symlink_path)
+    def symlink_latest(self) -> None:
```

Comment:
I added this as a function to the Summarizer class to leverage self.run_release_path (otherwise, feeding in the 'suite' is a bit more complicated since you have to specify if you're using a suite or a release and pass the variable in accordingly, so this makes it slightly easier)